src/pages/home/Home.js

```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from "react";
+
+// COMPONENTS
+import ByMonthColumnChart from"../../components/charts/ByMonthColumnChart";
+import AllMonthColumnChart from"../../components/charts/AllMonthColumnChart";
+
+const Home = () => {
+  return (
+    <Fragment>
+      <ByMonthColumnChart year={2016} />
```

Comment:
The measures/attributes/viewBy(s) should be calculated in this `Home` component and then passed to the children components.